modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -405,8 +399,7 @@ export default {
     async completeUploading (imgIds) {
       this.currentPage = 1;
       this.items = [];
-      await this.getMedia();
-
+      await this.debouncedGetMedia.skipDelay();
```

Comment:
Invoking the debounced version but skipping the delay keeps all the benefits of race condition prevention.